wecs/panda3d/character.py

```diff
@@ -78,14 +81,19 @@ class InertialMovement:
     ignore_z: bool = True
 
 
+@Component()
+class TurningBackToCameraMovement:
+    view_axis_allignment: float = 1
+
+
 @Component()
 class BumpingMovement:
     tag_name: str = 'bumping'
     solids: dict = field(default_factory=lambda:dict())
     contacts: list = field(default_factory=list)
     traverser: CollisionTraverser = field(default_factory=CollisionTraverser)
     queue: CollisionHandlerQueue = field(default_factory=CollisionHandlerPusher)
-    debug: bool = False
+    debug: bool = True
```

Comment:
Not as defaults. If you want debug, please set that in `main.py`. Next step then would be to stuff them into variables, and load them from a local config file. I'll put that on my TODO, but please don't make all games run in debug mode by default.